lib/identity_cache/query_api.rb

```diff
@@ -74,6 +74,32 @@ def fetch_multi(*ids)
         records
       end
 
+      def prefetch_associations(associations, records)
+        return if records.empty?
+
+        case associations
+        when nil
```

Comment:
this is an intentional do-nothing case? would be nice to have a comment making that explicit, otherwise people might think it was supposed to fall-through